myhpi/core/templates/core/minutes.html

```diff
@@ -25,6 +25,11 @@ <h4>Participants</h4>
                     <li>{{ participant.get_full_name }}</li>
                 {% endfor %}
             </ul>
+            {%  if page.labels.all %}
```

Comment:
maybe we want to add an heading here? 1327 reads "Labels" above the actual labels

Reply:
additional wish: only render the headline if there are labels

Reply:
1327 does something different:
![image](https://user-images.githubusercontent.com/6863832/178235577-88f25e45-074f-4819-8502-b94ac3b43bf8.png)

What is it?

Reply:
Then approve please

Reply:
i already did at 11:16